libbeat/processors/dissect/testdata/dissect_tests.json

```diff
@@ -130,11 +130,10 @@
 	},
 	{
 		"name": "indirect field",
-		"tok": "%{key} %{\u0026key}",
+		"tok": "%{?key} %{\u0026key}",
 		"msg": "hello world",
 		"expected": {
 			"hello": "world",
-			"key": "hello"
 		},
```

Comment:
This become invalid with the new validation that enforce strict 1:1 for reference and indirect field.